packages/venia-ui/lib/components/AddressBookPage/addressCard.css

```diff
@@ -5,13 +5,43 @@
     display: grid;
     grid-template-columns: 1fr max-content;
     padding: 1.25rem 2rem;
+    position: relative;
 }
 
 .root_updated {
     composes: root;
     animation: flash var(--venia-global-anim-bounce) 640ms 2;
 }
 
+.confirmDeleteContainer {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(255, 255, 255, 0.9);
```

Comment:
See if we can use a token for this.

Reply:
I'll leave this for a future use case. Let us slide it for this PR though.